Modules/ModParser.lua

```diff
@@ -1317,6 +1317,9 @@ local function extraSkill(name, level, noSupports)
 		}
 	end
 end
+local function upperCaseFirstLetter(str)
+	return (str:gsub("^%l", string.upper))
```

Comment:
Style: remove unnecessary parentheses

```suggestion
	return str:gsub("^%l", string.upper)
```

I might also suggest to use this function in a couple more places, like lines 824 and 827.  They use a slightly different pattern, but it's identical in usage.

Reply:
This is intentionally done so that Lua only returns one value, gsub returns two values by default (the string and the number of times it did substitutions).

See https://www.luafaq.org/gotchas.html#T8.1

Reply:
Ah, neat, didn't know about that.  Disregard me, then!